src/app/common/pipes/event-type-filter.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { MetadataBlockFragment } from "src/app/api/kamu.graphql.interface";
+
+@Pipe({
+    name: "eventTypeFilter",
+})
+export class EventTypeFilterPipe implements PipeTransform {
+    transform(
+        blocks: MetadataBlockFragment[],
+        filter: string,
```

Comment:
Think of supporting multiple filters simultaneously.

Reply:
Yes, added ng-multiselect-dropdown library